src_plugins/create/ppx_deriving_create.cppo.ml

```diff
@@ -1,11 +1,9 @@
-#include "../compat_macros.cppo"
-
-open Longident
```

Comment:
`Longident` is not opened anymore. @thierry-martinez explains that we could not open Ppxlib.Longident in the global scope because that one shadows `(=)` from the standard library (to be monomorphic on longidents), which we use in plugin code.